From 6f9c011f043bcea30093f0d12a5bbc111988707e Mon Sep 17 00:00:00 2001
From: Joe Hirschfeld <j@ibj.io>
Date: Sat, 2 Dec 2023 14:08:47 -0800
Subject: [PATCH 4/6] Explicitly ignore the return value of v4l2-ctl
 --list-devices

We can probably just ignore this. When packaging for nix this will probably not work correctly but this at least gets it to compile.
---
 device/camera/camera_input.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/device/camera/camera_input.c b/device/camera/camera_input.c
index b5632aa..759e4b4 100644
--- a/device/camera/camera_input.c
+++ b/device/camera/camera_input.c
@@ -19,7 +19,7 @@ static int camera_configure_input_v4l2(camera_t *camera)
   camera->camera = device_v4l2_open(camera->name, path);
   if (!camera->camera) {
     LOG_INFO(camera, "Listing available v4l2 devices:");
-    system("v4l2-ctl --list-devices");
+    (void) !system("v4l2-ctl --list-devices");
     return -1;
   }
 
-- 
2.39.3 (Apple Git-145)

